### ISSUE TITLE
Screen reader is not announcing 'No data to display' information when focus lands on chart when their is no data.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#20935]
#### Expected behaviour
<html>
<body>

If there is 'No data' Present on the graph, the screen reader should announce 'No data to display' information along with the graph information when focus lands on graph.
--

</body>
</html>

#### Actual behaviour
Screen reader is not announcing 'No data to display' information when focus lands on chart when their is no data.

#### Live demo with steps to reproduce

1.Open the URL [U.S Solar Employment Growth (codepen.io)](https://codepen.io/soqnwimw-the-selector/pen/WNWebLK).
2.U.S Solar Employment Growth page will appear navigate below the code Section.
3.Navigate inside U.S Solar Employment Growth section to and observe the screen reader announcement.

https://github.com/highcharts/highcharts/assets/93735775/9b73da07-098a-42ed-8f1e-495e816dbb20

#### Product version

Version 9.0.0 of Highcharts

#### Affected browser(s)
Edge

**Comment by karolkolodziej** [comment]
Hi @msftedad! 
Thank you for describing the issue. 

Have you tried a later version of HIghcharts? It works slightly differently now.
Please test that and let me know if this works for you.

**Comment by msftedad** [comment]
Hi @karolkolodziej, Currently we are checking this issue with latest version. we have closed this issue for now. If we face any issue further we will connect with you. Thanks
